wren-modeling-rs/sqllogictest/test_sql_files/model.slt

```diff
@@ -0,0 +1,2 @@
+statement ok
+SELECT * from orders
```

Comment:
This test should be `SELECT * from wrenai.default.orders` actually.
There is an issue about analyzing a qualified table name. I'll address this in next PR.